addons/spreadsheet/static/tests/lists/list_plugin.test.js

```diff
@@ -394,6 +406,67 @@ test("Can undo/redo a delete list", async function () {
     expect(B4.value).toBe("#ERROR");
 });
 
+test("can update a list", async () => {
+    let spreadsheetLoaded = false;
+    let isInitialUpdate = true;
+    const { model } = await createSpreadsheetWithList({
+        mockRPC: async function (route, args) {
+            if (
+                spreadsheetLoaded &&
+                args.method === "web_search_read" &&
+                args.model === "partner"
+            ) {
+                expect.step("data-fetched");
+                if (isInitialUpdate) {
```

Comment:
Could be worth checking the order and domain during the undo/redo.

Reply:
good catch, thanks!